contracts/VaultManager/VaultManager.sol

```diff
@@ -34,6 +34,21 @@ contract VaultManager is IVaultManager, VaultManagerEvents,GovManager{
         TokenToVaultId[_tokenAddress] = vaultId;
         emit NewVaultCreated(vaultId, _tokenAddress);
     }
+    
+    function DeleteVault(address _tokenAddress)
+        external
+        override
+        onlyOwnerOrGov
+        vaultExists(TokenToVaultId[_tokenAddress])
+        returns (uint vaultId)
+    {
+        Vault vault = Vault(VaultIdToVault[TokenToVaultId[_tokenAddress]]);
+        require(vault.tokenBalance() == 0, "VaultManager: Vault not empty");
+        vaultId = TokenToVaultId[_tokenAddress];
+        delete VaultIdToVault[vaultId];
+        delete TokenToVaultId[_tokenAddress];
+        emit VaultDeleted(vaultId, _tokenAddress);
```

Comment:
I think we need just to remove the option to create new pools on that vault, 
the money stored in the vault -can still be used for withdrawal.

Reply:
In the vault's language, you mean to just pause deposits on that vault while keeping the withdrawals active, right? How about making new state variables  `isDepositeActiveFor[vaultId]` and `isWithdrawablActiveFor[vaultId]`?

Reply:
I feel from all roles should be managed by Governor. Vault only needs one admin, like other all contracts.